bee-node/docker/bee.yaml

```diff
@@ -0,0 +1,19 @@
+version: "3.9"
+services:
+  bee:
+    build:
+      context: ../../
+      dockerfile: ./bee-node/docker/Dockerfile
+    image: bee:${IMAGE_TAG}
+    working_dir: /bee_workdir
+    tty: true # for colored logs
+    container_name: "bee"
+    ports:
+      - "14265:14265"
+      - "15600:15600"
+      - "1883:1883"
+      - "8081:8081"
+    volumes:
+      - ../config.toml:/bee_workdir/config.toml
+      - ../storage:/bee_workdir/storage
+    # restart: always
```

Comment:
Is this to be removed?

Reply:
I could remove it, I commented just to show that if someone want to run the node indefinitely it could uncomment that line, so let me know if it could be usefull or jut to delete it